pkg/service/system/route53.go

```diff
@@ -129,3 +182,194 @@ func AddRoute53Record(ctx context.Context, dnsName, recordName, regionName strin
 
 	return *result.ChangeInfo.Id, nil
 }
+
+// GetRoute53TXTRecords will return TXT records only
+func GetRoute53TXTRecords(ctx context.Context) ([]types.Route53Record, error) {
+
+	hostedZoneId := config.Get().AwsRoute53HostedZoneID
+
+	// Creating AWS Route53 session
+	route53Client, err := getRoute53Sess(defaultRegion)
```

Comment:
in some places like create(), we are using region in request, how is this default region ? Are we expecting to use in case of region missing or is the fixed region we will be using. cc; @sid255 
lets move this to config either way, looks variable to me, wdyt ?